### DISCUSSION TITLE
getting error messages

### DISCUSSION THREAD

**Discussion by Glycode** [negue/meme-box#477] [Q&A]
so I resetted my windows and also wanted to setup memebox from scratch. I used the beta version but ran into a problem so I went back to an earlier version of memebox.
But I keep getting error messages when I want to delete stuff or open the config folder

![image](https://user-images.githubusercontent.com/97413783/181916217-ca64236c-77b4-4897-855c-7b68508382d2.png)

![image](https://user-images.githubusercontent.com/97413783/181916197-bdb67560-5d42-440a-84f3-844d7ed8496b.png)

is it due to overlapping of some files from the different versions? How can I reset it?

**Comment by negue** [comment]
Hey,

on `focusInitialElementWhenReady`  I need to search where is coming from - because I don't have atleast in my code, I'll respond once I found for that.

on `open/config` this seems the Memebox UI you are seeing is still from an older Version but the Server-Side is already updated  (because its now api/open/config ) so I guess you just need to Strong-Refresh once usually (CTRL + F5) and then retry open the Config Folder 


I have cache-checks in place that should've prevented such behavior - but I guess I need to try add more of these in ... there is always something broken :D 


but re-reading that - I don't know how well a downgrade using the same config works for that you definitly need to delete everything (from memembox)   - for the next time - while its still in beta - you could use the following command line arguments https://github.com/negue/meme-box/blob/develop/tutorials/installation.md#commandline-options    so that the new version is using a different config path so that you aren't breaking your current one


as for the `I used the beta version but ran into a problem`  is the other discussion that you opened? or some complete different issue?

**Comment by Glycode** [comment]
I tried deleting everything from memebox (everything I could find in appdata). But apparently that wasnt enough. 

How do I use the strong refresh (or where) because nothing happens in memebox.

How or where do i use the command line?

No the issue from the other thread was a different one

**Reply by negue** [reply]
> I tried deleting everything from memebox (everything I could find in appdata). But apparently that wasnt enough.

ok thats very odd then o.O

> How do I use the strong refresh (or where) because nothing happens in memebox.

if you have it in the electron version - Pressing `ctrl` and `F5` should do the trick

> How or where do i use the command line?

you could create a link of the exe and then apply the command line arguments there or - create a *.bat File which has something like 

```
memebox.exe --config=Path/To/The/Config/Directory
```

what was the other issue? cause I'd need to fix that anyway since the new version will the "soon" the one that is replaced
